app/src/main/java/com/kyphipraniti/fitnesstasks/adapters/RVAdapter.java

```diff
@@ -0,0 +1,73 @@
+package com.kyphipraniti.fitnesstasks.adapters;
+
+import java.util.ArrayList;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.kyphipraniti.fitnesstasks.R;
+import com.kyphipraniti.fitnesstasks.activities.WorkoutDetailActivity;
+
+/**
+ * Created by praniti on 10/11/17.
+ */
+
+public class RVAdapter extends RecyclerView.Adapter<RVAdapter.MyViewHolder> {
+
+    ArrayList<String> workoutNames;
+    ArrayList<Integer> workoutImages;
+    Context context;
+
+    public RVAdapter(Context context, ArrayList workoutNames, ArrayList workoutImages) {
+        this.context = context;
+        this.workoutNames = workoutNames;
+        this.workoutImages = workoutImages;
+    }
+
+    @Override
+    public MyViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View v = LayoutInflater.from(parent.getContext()).inflate(R.layout.rowlayout, parent, false);
+        MyViewHolder vh = new MyViewHolder(v);
+        return vh;
+    }
+
+    @Override
+    public void onBindViewHolder(MyViewHolder holder, final int position) {
+
+        holder.name.setText(workoutNames.get(position));
+        holder.image.setImageResource(workoutImages.get(position));
+        holder.itemView.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Intent intent = new Intent(context, WorkoutDetailActivity.class);
+                intent.putExtra("image", workoutImages.get(position));
+                context.startActivity(intent);
+            }
+        });
+    }
+
+    @Override
+    public int getItemCount() {
+        return workoutNames.size();
+    }
+
+    public class MyViewHolder extends RecyclerView.ViewHolder {
```

Comment:
could you rename this to just ViewHolder

Reply:
done